playground/nitro.config.ts

```diff
@@ -2,4 +2,5 @@ import { defineNitroConfig } from "nitropack/config";
 
 export default defineNitroConfig({
   compatibilityDate: "2024-09-29",
+  preset: 'coolify',
```

Comment:
Please do not modify playground. You can build with `nitro build --preset coolify` to test.

Reply:
Removed the modification to the playground, thank you for the advice!